app/views/external_users/claims/expenses/_expense_fields.html.haml

```diff
@@ -1,134 +1,110 @@
-
+          - expense_types_collection = present_collection(ExpenseType.for_claim_type(@claim))
+          = f.govuk_collection_select :expense_type_id,
+            expense_types_collection.map { |et| [et.name, et.id, { data: et.data_attributes }] },
+            :second,
+            :first,
+            form_group: { classes: 'fx-travel-expense-type' },
+            label: { text: t('.type_of_expense_html') },
+            options: { include_blank: t('.select_option') }
+
+          - reasons_collection = present_collection(reasonset_for_expense_type(f.object.expense_type).values, TravelReasonPresenter)
+          = f.govuk_collection_select :reason_id,
+            reasons_collection.map { |et| [et.reason, et.id, { data: et.data_attributes }] },
+            :second,
+            :first,
+            form_group: { classes: 'fx-travel-reason', style: 'display: none' },
+            label: { text: t('.travel_reason_html') },
+            options: { include_blank: t('.select_option') }
+
+          - present(f.object) do |expense|
+            = f.govuk_text_field :reason_text,
+              form_group: { classes: 'fx-travel-reason-other', style: f.object.reason_id.eql?(5) ? 'display: block' : 'display: none' },
+              label: { text: t('.reason_text_html') },
+              width: 'one-half'
+
+          .fx-travel-location{ style: 'display: none' }
             - if @claim.lgfs?
-              - error_classes = error_class?(@error_presenter, "expense_#{@expense_count}_location")
-              - error_classes += ' form-group-error' if error_classes.present?
-              .form-group.fx-establishment-select.has-select{ style: 'display: none;', class: error_classes }
-                = f.label :location, nil, class: 'form-label-bold'
-                = validation_error_message(@error_presenter, "expense_#{@expense_count}_location")
-                = f.select :location, {}, {}, { class: 'form-control form-control-3-4', autocomplete: 'off' }
-
-            = f.adp_text_field :location, label: t('.location_html'), input_classes: 'form-control form-control-3-4 fx-location-model', errors: @error_presenter
-
-        .govuk-grid-column-full
-          -# INPUT: Travel hours
-          .form-group.fx-travel-hours{ style: 'display: none;' }
-            = f.adp_text_field :hours, label: t('.hours_html'), input_classes: 'form-control-1-8', input_type: 'number', errors: @error_presenter
+              = f.govuk_collection_select :location,
+                {},
+                {},
+                {},
+                form_group: { classes: 'fx-establishment-select has-select', style: 'display: none' },
+                label: { text: nil }
+
+            = f.govuk_text_field :location,
+                class: 'fx-location-model',
+                form_group: { classes: 'location_wrapper' },
+                label: { text: t('.location_html') },
+                width: 'one-half'
+
+          = f.govuk_number_field :hours,
```

Comment:
Nit: consider adding a `min:` value to  all `govuk_number_field` to limit errors